BUILD.md

```diff
@@ -737,10 +737,6 @@ Tested on OSX version 10.13.3
 
 Setup Homebrew and components
 
-- Follow instructions on [brew.sh](http://brew.sh) to get Homebrew installed.
-
-      /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-
```

Comment:
These instructions are out of date. Trying to run this won't work now.

https://raw.githubusercontent.com/Homebrew/install/master/install